.orchestra/ci/ci-hook/app.py

```diff
@@ -63,12 +121,24 @@ def get_env_or_fail(var_name):
     - private: git@rev.ng:revng-private/binary-archives.git
 """)
 
+pull_request_user_options = dedent("""
```

Comment:
Is this actually pull-request specific?

Can you recap for me what webhooks are we listening to?
I understand we're no longer listening to push webhooks, in favor of checks webhooks. But AFAIU those are not pull-request specific, so maybe we can have a more appropriate name?

Reply:
GH:
- pull_request:
  - opened
  - synchronize (push)
- check_run/check_suite:
  - requested (push to the main repo)
  - rerequested (user clicks re-run)

GL:
- Push
- Merge Request

> so maybe we can have a more appropriate name?

I'd say just `user_options`, it's used always except when pushing